docs/docs/cmd/teams/meeting/meeting-transcript-list.md

```diff
@@ -0,0 +1,93 @@
+# teams meeting transcript list
+
+Lists all transcripts for a given meeting.
+
+## Usage
+
+```sh
+m365 teams meeting transcript list [options]
+```
+
+## Options
+
+`-u, --userId [userId]`
+: The id of the user, omit to list meeting transcripts list for current signed in user. Use either `id`, `upn` or `email`, but not multiple.
+
+`--upn [upn]`
```

Comment:
Instead of upn, this should be `userName` as mentioned in the specs. I think you misunderstood my last remark 😅. I meant to clarify the description, not change the option name 😄

Reply:
🤦‍♂️

Reply:
@milanholemans not that I don't want to revert it 😉 but TBH what do you think we actually leave this `upn`? I mean if it was not clear what `userName` was and we need to clearly state it in the description why not just rename it to `upn` so what I did? I mean it may not be clearer than that right 🙂🤔
also we already use `upn` options in some `spo` commands so it would not be something totally new I guess 🤔

Reply:
I'm not the biggest fan of `userName` as option, but this is something we use almost everywhere. If you check the other `teams meeting` commands, they all use `userName`, same goes to `aad user` commands etc. So for the sake of consistency, I'd use here `userName` as well.

Reply:
ok if it will be consistent with other teams commands then I agree 👍
I wonder if we should align and use either `userName` or `upn` everywhere (as I know the second is used in many places on `spo`)  but this is a different story (issue) 😉

Reply:
Seems like 39 commands use `userName` and 10 use `upn`. Aligning these would be a good idea indeed. Consistency is quite important.

Reply:
I agree 😉 
so on whose 'todo' list is it now 🤔 Yours or mine 😅😜?

Reply:
The honor is yours 😄

Reply:
but by all means, I do not whish to steal that honor from you.  😅

Reply:
✅